### ISSUE TITLE
508-defect-1 ❗ Launchblocker [COGNITION, FOCUS MGMT]: Upon completion of processing the form, focus MUST announce the context change

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12691]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

After submitting the form using NVDA and Firefox, I heard "processing" your application, but when it completed there was no announcement to the screen reader. The context change **must** be communicated to assistive tech, and this could be done by moving focus to the "Your claim has successfully been submitted". 

This raises another issue, that the "Please print this page for your records" should come after the announcement about the submission status, because a screen reader user wouldn't get the information to print the page.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to know when my form has been processed and what the status is.

## Environment

* Operating System: Windows (but I think this would happen in all; I waited to submit the application until I did the last screen reader)
* Browser: Firefox (would happen in all)
* Screenreading device: NVDA (would happen in all)
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/disability/file-disability-claim-form-21-526ez/introduction in browser
1. Start your screen reader
1. Log in (I used user 261), complete the form, and submit
1. Verify that you hear that the form is processing
1. Verify that you are not informed when the submission is complete

## Possible Fixes (optional)

1. Use JavaScript to move focus to the notice of submission status once the processing is complete.
1. In addition, move the "Please print this page for your records" inside the submission status alert box, following the notice.

## WCAG or Vendor Guidance (optional)

* [W3C - Name, Role, Value](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html) - 4.1.2 Name, Role, Value: For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and **notification of changes to these items is available to user agents, including assistive technologies**. (Level A)

## Screenshots or Trace Logs

![image (5)](https://user-images.githubusercontent.com/57469/90828585-47af3980-e30c-11ea-9a44-9bb23d01c03e.png)

**Comment by Mottie** [comment]
Fix has been merged into master. It should be available on staging momentarily.

**Comment by jenstrickland** [comment]
Validated 🎉  ☘️
